indico/modules/events/layout/controllers.py

```diff
@@ -78,12 +116,29 @@ def _process(self):
         f = request.files.get('file')
         filename = secure_filename(f.filename, 'stylesheet')
         content_type = mimetypes.guess_type(f.filename)[0] or f.mimetype or 'application/octet-stream'
+        existing_css_file = StylesheetFile.find_first(event_id=self._conf.id)
```

Comment:
StylesheetFile.find(event_id=self._conf.id).delete()

not sure if it causes any inconsistencies, but it shouldn't since no other objects reference it